clean-blog/templates/base.html

```diff
@@ -137,23 +135,12 @@
             <div class="row">
                 <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                     <p class="copyright text-muted">All content <a href="https://creativecommons.org/licenses/by-sa/4.0/">CC by SA</a> unless stated otherwise.</p>
-                    <p class="copyright text-muted">Blog powered by <a href="http://getpelican.com">Pelican</a>,
```

Comment:
I'm wondering why we are removing the credits to the tools we are using?

Reply:
oops, that was not on purpose...